sources/ghostdog/src/main.rs

```diff
@@ -63,6 +141,44 @@ fn run() -> Result<()> {
             let device_name = find_device_name(format!("{}", path.display()))?;
             emit_device_name(&device_name);
         }
+        SubCommand::MatchNvidiaDriver(vendor) => {
+            // Currently, we only want to run this for NVIDIA devices
+            if vendor.vendor == NVIDIA_VENDOR_ID {
```

Comment:
This seems redundant with the udev rule, which only invokes `ghostdog match-nvidia-driver` if the vendor ID matches.